spdx/package.py

```diff
@@ -192,7 +192,12 @@ def calc_verif_code(self):
         hashes = []
 
         for file_entry in self.files:
-            hashes.append(file_entry.calc_chksum())
+            if isinstance(file_entry.chk_sum, checksum.Algorithm) and
```

Comment:
Thanks. I probably need to work on #4 next so I get slapped for blindly typed stuff like that :-D